resources/views/layouts/partials/header/autocomplete.blade.php

```diff
@@ -17,7 +17,7 @@
         <ais-instant-search
             class="contents"
             :search-client="searchClient"
-            :index-name="config.index_prefix + '_products_' + config.store"
+            index-name="{{ (new (config('rapidez.models.product')))->searchableAs() }}"
```

Comment:
This is now repeated multiple times; can't we just push this in the config and use that? `config.index.products`, `config.index.categories`, etc. as mentioned in the ConfigComposer? And please double check that file, search for `index`

Reply:
Fixed in #769